resolwe_bio/processes/workflows/chip_seq.yml

```diff
@@ -798,3 +798,237 @@
         run: multiqc
         input:
           data: '{{ [input.case, steps.macs2, steps.chipqc] }}'
+
+- slug: workflow-subsample-bwa-aln-single
+  name: "Subsample FASTQ and BWA Aln (single-end)"
+  data_name: "{{ reads|sample_name|default('?') }}"
+  requirements:
+    expression-engine: jinja
+  version: 1.0.0
+  type: data:workflow:chipseq:seqtkbwaaln
+  category: Pipeline
+  input:
+    - name: reads
+      label: Reads
+      type: data:reads:fastq:single
+    - name: genome
+      label: Reference genome
+      type: data:index:bwa
+    - name: downsampling
+      label: Downsampling
+      group:
+        - name: n_reads
+          label: Number of reads
+          type: basic:integer
+          default: 10000000
+        - name: advanced
+          label: Advanced options
+          group:
+            - name: seed
+              label: Seed
+              type: basic:integer
+              default: 11
+            - name: fraction
+              label: Fraction
+              type: basic:decimal
+              required: false
+              range: [0, 1.0]
+              description: |
+                Use the fraction of reads [0 - 1.0] from the orignal input file instead
+                of the absolute number of reads. If set, this will override the
+                "Number of reads" input parameter.
+            - name: two_pass
+              label: 2-pass mode
+              type: basic:boolean
+              default: true
+              description: |
+                Enable two-pass mode when down-sampling. Two-pass mode is twice
+                as slow but with much reduced memory.
+    - name: alignment
+      label: alignment
+      group:
+        - name: q
+          label: Quality threshold
+          type: basic:integer
+          default: 5
+          range: [0, 60]
+          description: |
+            Parameter for dynamic read trimming.
+        - name: use_edit
+          label: Use maximum edit distance (excludes fraction of missing alignments)
```

Comment:
I would move the text in the brackets into description and perhaps expand it to include more information on what this option is.

Reply:
I would leave rewording of the descriptions for another time, since I think this is beyond the scope of this PR for several reasons:
1) All of those descriptions are copied from other (existing) processes/workflows and they would need to be updated as well.
2) in many cases, the parameter description is copied directly from the underlying tool documentation, which itself could often be much better.
3) Its often hard to parse the actual meaning of the parameters in use, even if you inspect the tool documentation directly. The "edit distance" above comes from [Levenshtein_distance](https://en.wikipedia.org/wiki/Levenshtein_distance) but it takes time to find this kind of proper descriptions and reword them, which again, is beyond the scope of this PR.